crates/cheatcodes/spec/src/vm.rs

```diff
@@ -609,6 +609,14 @@ interface Vm {
     #[cheatcode(group = Evm, safety = Unsafe)]
     function coolSlot(address target, bytes32 slot) external;
 
+    /// Returns the execution evm version.
```

Comment:
actually can we add a note on the cheatcodes docs too? that runtime/execution evm version is not the same as compilation one